apps/gp2-server/src/validation/output.validation.ts

```diff
@@ -100,6 +100,10 @@ const outputPostRequestValidationSchema: JSONSchemaType<gp2Model.OutputPostReque
         type: 'string',
         nullable: true,
       },
+      shortDescription: {
+        type: 'string',
+        nullable: true,
```

Comment:
what about maximum length?

Reply:
nice catch, I've added it here [add missing maxLength validation](https://github.com/yldio/asap-hub/pull/4242/commits/0603e2d1f314d630fb06b45c544ef58193e2426d)